runtime/runtime-core/jvm/src/aws/smithy/kotlin/runtime/time/InstantJVM.kt

```diff
@@ -40,7 +41,7 @@ actual class Instant(internal val value: jtInstant) : Comparable<Instant> {
      * Encode the [Instant] as a string into the format specified by [TimestampFormat]
      */
     actual fun format(fmt: TimestampFormat): String = when (fmt) {
-        TimestampFormat.ISO_8601 -> ISO_INSTANT.format(value)
+        TimestampFormat.ISO_8601 -> ISO_INSTANT.format(value.truncatedTo(ChronoUnit.MICROS))
```

Comment:
is there any downside to always defaulting to microseconds? assuming not but just asking

Reply:
The downside is loss of resolution but there is going to be no way to tell which services support finer grained precision.

Reply:
I figured there was no way to tell